src/extension.ts

```diff
@@ -62,6 +62,8 @@ export function activate(context: vscode.ExtensionContext): void {
     ext.outputChannel = outputChannel;
     context.subscriptions.push(outputChannel);
 
+    const logsOutputChannels: { [channelName: string]: { channelObj: vscode.OutputChannel } } = {};
```

Comment:
I still think this should be `vscode.OutputChannel | undefined` as described in my other comment, but I also don't think `channelObj` is necessary. You can just do: `{ [channelName: string]: vscode.OutputChannel | undefined }`